src/main/java/test/CarsTableTest.java

```diff
@@ -19,61 +19,73 @@
 public class CarsTableTest {
     private Connection connection;
     private CarsTableManagementService carsTableManagementService;
+
+    private final String tableName = "cars";
```

Comment:
tableName może tu zostać tylko daj static i zmien nazwę CARS_TABLE_NAME.
Reszta tych pól to raczej dane testowe i bez sensu je wyciągać, mogą być od razu w metodzie

Reply:
sonar do tego sie czepial, dlatego wyciagnalem. Zmienie tak jak mowisz